ocs_ci/ocs/ui/base_ui.py

```diff
@@ -836,8 +837,24 @@ def take_screenshot(driver):
     time.sleep(0.5)
 
 
-@retry(TimeoutException, tries=3, delay=3, backoff=2)
-@retry(WebDriverException, tries=3, delay=3, backoff=2)
+def garbage_collector_webdriver():
+    """
+    Garbage Collector for webdriver objs
+
+    """
+    collected_objs = gc.get_objects()
+    for obj in collected_objs:
+        if str(type(obj)) == "<class 'selenium.webdriver.chrome.webdriver.WebDriver'>":
```

Comment:
@OdedViner can we move the "<class 'selenium.webdriver.chrome.webdriver.WebDriver'>" string to constants.py?

Reply:
done